retroshare-gui/src/gui/qss/stylesheet/Standard.qss

```diff
@@ -319,20 +319,29 @@ GenCertDialog > QFrame#headerFrame > QLabel#headerLabel {
 /* ConnectFriendWizard */
 
 ConnectFriendWizard QWizardPage#ConclusionPage > QGroupBox#peerDetailsFrame {
-	border: 2px solid #CCCCCC;
+	border: 2px solid #039bd5;
 	border-radius:6px;
 	background: white;
 	padding: 12 12px;
 }
 
-ConnectFriendWizard QLabel#fr_label, QLabel#requestinfolabel
+ConnectFriendWizard QLabel#fr_label, QLabel#requestinfolabel, QLabel#cp_Label
 {
 	border: 1px solid #DCDC41;
 	border-radius: 6px;
 	background: #FFFFD7;
 	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFFFD7, stop:1 #FFFFB2);
 }
 
+ConnectFriendWizard QGroupBox::title#peerDetailsFrame
+{
+	padding: 4 12px;
+	background: transparent; 
+    padding: 4 12px;
+    background: #039bd5;
+    color: white;
+}
+
```

Comment:
Don't forget other qss files.

Reply:
which other? there is no other standard qss files for the default style, 
i dont do changes on other external stylesheets, where nobody works more on it

Reply:
These ones:
https://github.com/RetroShare/RetroShare/tree/master/retroshare-gui/src/qss
Nobody works on it because every body miss them...

Reply:
yes i has no time for this, i hoped that some one else can improve work on the stylesheets